### DISCUSSION TITLE
tvOS homepage refresh

### DISCUSSION THREAD

**Discussion by rwjack** [jellyfin/Swiftfin#1220] [Questions] [answered]
Hey, I'm seeing major progress on both the iOS and tvOS end (through the commit history), but I can't tell if this feature has been implemented yet.

Do we have an update for testflight on tvOS? It's been ages since we've had an update, although small incremental releases would be far better than where we're at right now.

**Comment by siddharthgpta** [comment]
Yeah I checked the version of the app on tvOS and it’s on 1.0.1 which was released in March 2023. The most recent version of the app on the phone is 1.1.1 which is about 2 months old. 

I’ve just been contemplating whether I should build it myself and sideload it in dev mode or something.

**Comment by JPKribs** [comment, marked as answer]
Hey all,

There is work being done for tvOS and a lot of issues that exist in 1.0.1 (the current App Store build) are resolved on the Main/Development branch of Swiftfin. That being said, the current state of that build is missing some key features that we need to address before we can proceed.

I've compiled everything we are working on to get this build ready for the App Store here:

https://github.com/jellyfin/Swiftfin/discussions/1294

If you're interested in following our progress, I will be updating that post as things change!
